`LongPollingTransportOnReceiveGetsCalled()` is flaky

## Failing Test(s)

<!--
Provide the fully qualified name(s) of the failing tests.
-->

- `com.microsoft.signalr.LongPollingTransportTest.LongPollingTransportOnReceiveGetsCalled()`

## Error Message

<!--
Provide the error message associated with the test failure, if applicable.
-->

```text
expected: <true> but was: <false>
```

## Stacktrace

<details>
<!--
Provide the stack trace associated with the test failure, if applicable.
-->

```text
org.opentest4j.AssertionFailedError: expected: <true> but was: <false>
	at org.junit.jupiter.api.AssertionUtils.fail(AssertionUtils.java:38)
	at org.junit.jupiter.api.AssertTrue.assertTrue(AssertTrue.java:40)
	at org.junit.jupiter.api.AssertTrue.assertTrue(AssertTrue.java:35)
	at org.junit.jupiter.api.Assertions.assertTrue(Assertions.java:137)
	at com.microsoft.signalr.LongPollingTransportTest.LongPollingTransportOnReceiveGetsCalled(LongPollingTransportTest.java:159)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.junit.platform.commons.util.ReflectionUtils.invokeMethod(ReflectionUtils.java:515)
	at org.junit.jupiter.engine.execution.ExecutableInvoker.invoke(ExecutableInvoker.java:115)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.lambda$invokeTestMethod$6(TestMethodTestDescriptor.java:171)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:72)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.invokeTestMethod(TestMethodTestDescriptor.java:167)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.execute(TestMethodTestDescriptor.java:114)
	at org.junit.jupiter.engine.descriptor.TestMethodTestDescriptor.execute(TestMethodTestDescriptor.java:59)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$5(NodeTestTask.java:105)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:72)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:95)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:71)
	at java.util.ArrayList.forEach(ArrayList.java:1257)
	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.invokeAll(SameThreadHierarchicalTestExecutorService.java:38)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$5(NodeTestTask.java:110)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:72)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:95)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:71)
	at java.util.ArrayList.forEach(ArrayList.java:1257)
	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.invokeAll(SameThreadHierarchicalTestExecutorService.java:38)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$executeRecursively$5(NodeTestTask.java:110)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:72)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.executeRecursively(NodeTestTask.java:95)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:71)
	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.submit(SameThreadHierarchicalTestExecutorService.java:32)
	at org.junit.platform.engine.support.hierarchical.HierarchicalTestExecutor.execute(HierarchicalTestExecutor.java:57)
	at org.junit.platform.engine.support.hierarchical.HierarchicalTestEngine.execute(HierarchicalTestEngine.java:51)
	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:170)
	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:154)
	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:90)
	at org.junit.platform.console.tasks.ConsoleTestExecutor.executeTests(ConsoleTestExecutor.java:65)
	at org.junit.platform.console.tasks.ConsoleTestExecutor.lambda$execute$0(Co
```
</details>


## Logs

<details>
<!--
Provide the (helix) logs associated with the test failure, if applicable.
-->

There is a lot of information in https://helixre8s23ayyeko0k025g8.blob.core.windows.net/dotnet-aspnetcore-refs-heads-main-a61547be830e450ca0/signalr.client.java.Tests--/console.9d51f993.log?sv=2019-07-07 but it's not clear what's relevant. Copying a little bit here.

```text
09:27:51.476 [pool-175-thread-1] INFO com.microsoft.signalr.HubConnection - HubConnection started.
09:27:51.477 [pool-175-thread-1] DEBUG com.microsoft.signalr.LongPollingTransport - OnReceived callback has been invoked.
09:27:51.477 [main] DEBUG com.microsoft.signalr.HubConnection - Stopping HubConnection.
09:27:51.477 [main] INFO com.microsoft.signalr.LongPollingTransport - LongPolling transport stopped.
09:27:51.477 [main] ERROR com.microsoft.signalr.HubConnection - HubConnection disconnected with an error Request has no handler: DELETE http://example.com?id=bVOiRPG8-6YiJ6d7ZcTOVQ.
09:27:51.478 [main] INFO com.microsoft.signalr.HubConnection - HubConnection stopped.
09:27:51.478 [main] DEBUG com.microsoft.signalr.HubConnection - The HubConnection is transitioning from the CONNECTED state to the DISCONNECTED state.
09:27:51.478 [RxCachedThreadScheduler-3] DEBUG com.microsoft.signalr.LongPollingTransport - Message received.
09:27:51.478 [RxCachedThreadScheduler-3] DEBUG com.microsoft.signalr.LongPollingTransport - Long Polling transport polling complete.
09:27:51.479 [main] DEBUG com.microsoft.signalr.HubConnection - The HubConnection is attempting to transition from the DISCONNECTED state to the CONNECTING state.
09:27:51.479 [main] DEBUG com.microsoft.signalr.HubConnection - Starting HubConnection.
09:27:51.480 [main] DEBUG com.microsoft.signalr.HubConnection - The HubConnection is attempting to transition from the CONNECTING state to the CONNECTED state.
09:27:51.480 [main] INFO com.microsoft.signalr.HubConnection - HubConnection started.
09:27:51.480 [Timer-130] DEBUG com.microsoft.signalr.HubConnection - Sending PING message.
09:27:51.480 [main] DEBUG com.microsoft.signalr.HubConnection - Stopping HubConnection.
09:27:51.481 [main] INFO com.microsoft.signalr.HubConnection - HubConnection stopped.
09:27:51.481 [main] DEBUG com.microsoft.signalr.HubConnection - The HubConnection is transitioning from the CONNECTED state to the DISCONNECTED state.
09:27:51.481 [main] DEBUG com.microsoft.signalr.HubConnection - The HubConnection is attempting to transition from the DISCONNECTED state to the CONNECTING state.
09:27:51.481 [main] DEBUG com.microsoft.signalr.HubConnection - Starting HubConnection.
09:27:51.482 [main] DEBUG com.microsoft.signalr.HubConnection - The HubConnection is attempting to transition from the CONNECTING state to the CONNECTED state.
09:27:51.482 [main] INFO com.microsoft.signalr.HubConnection - HubConnection started.
09:27:51.482 [main] DEBUG com.microsoft.signalr.HubConnection - Sending INVOCATION message '1'.
09:27:51.482 [Timer-131] DEBUG com.microsoft.signalr.HubConnection - Sending PING message.
09:27:51.483 [main] DEBUG com.microsoft.signalr.HubConnection - Received message of type COMPLETION.
09:27:51.486 [main] DEBUG com.microsoft.signalr.HubConnection - The HubConnection is attempting to transition from the DISCONNECTED state to the CONNECTING state.
09:27:51.487 [main] DEBUG com.microsoft.signalr.HubConnection - The HubConnection is attempting to transition from the CONNECTING state to the DISCONNECTED state.

Failures (1):
  JUnit Jupiter:LongPollingTransportTest:LongPollingTransportOnReceiveGetsCalled()
    MethodSource [className = 'com.microsoft.signalr.LongPollingTransportTest', methodName = 'LongPollingTransportOnReceiveGetsCalled', methodParameterTypes = '']
    => org.opentest4j.AssertionFailedError: expected: <true> but was: <false>

Test run finished after 7824 ms
[        17 containers found      ]
[         0 containers skipped    ]
[        17 containers started    ]
[         0 containers aborted    ]
[        17 containers successful ]
[         0 containers failed     ]
[       258 tests found           ]
[         0 tests skipped         ]
[       258 tests started         ]
[         0 tests aborted         ]
[       257 tests successful      ]
[         1 tests failed          ]


> Task :test:junitPlatformTest FAILED

FAILURE: Build failed with an exception.

* What went wrong:
Execution failed for task ':test:junitPlatformTest'.
> Process 'command '/Library/Java/JavaVirtualMachines/adoptopenjdk-8.jdk/Contents/Home/bin/java'' finished with non-zero exit value 1
```
</details>

## Build

<!--
Provide a link to the build where the test failure occurred.
-->

- https://dev.azure.com/dnceng/public/_build/results?buildId=1242525&view=results

--- a/src/SignalR/clients/java/signalr/test/src/main/java/com/microsoft/signalr/LongPollingTransportTest.java
+++ b/src/SignalR/clients/java/signalr/test/src/main/java/com/microsoft/signalr/LongPollingTransportTest.java
@@ -126,7 +126,6 @@ public class LongPollingTransportTest {
         assertTrue(onReceivedRan.get());
     }
 
-    @Disabled("https://github.com/dotnet/aspnetcore/issues/34563")
     @Test
     public void LongPollingTransportOnReceiveGetsCalled() {
         AtomicInteger requestCount = new AtomicInteger();

